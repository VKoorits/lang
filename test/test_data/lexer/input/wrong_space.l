if a > 10:
	print a
else:
# 5 space in next str
     print "a<10"

