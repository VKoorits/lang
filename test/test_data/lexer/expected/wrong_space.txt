ERROR: wrong count space in shift of string:
  5:	     print "a<10"

